src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 198px;
```

Comment:
It would be better to have max-width: 198px and width: 100%; so it can be narrower if needed

Reply:
> It would be better to have max-width: 198px and width: 100%; so it can be narrower if needed

Done